docs/customize/vocabularies/funding.md

```diff
@@ -31,7 +31,13 @@ Here is an example of two such records in YAML format:
       scheme: isni
   country: US
 - id: private-acme-funding
-  name: Private ACME Funding
+  name: National Institutes of Health
```

Comment:
Here we wanted to give clarity on the fact that identifiers are optional. Therefore, we only put the name. Was there any issues why this needs identifiers, country, etc.? If so we should put something that matches (not the ones from the NIH, from the example above).